hail/src/main/scala/is/hail/expr/types/MatrixType.scala

```diff
@@ -17,6 +18,12 @@ class MatrixTypeSerializer extends CustomSerializer[MatrixType](format => (
 object MatrixType {
   val entriesIdentifier = "the entries! [877f12a8827e18f61222c6c8c5fb04a8]"
 
+  def getRowType(rvRowType: PStruct): PStruct = rvRowType.selectFields(Set(entriesIdentifier), keep = false)
+  def getEntryArrayType(rvRowType: PStruct): PArray = rvRowType.field(entriesIdentifier).typ.asInstanceOf[PArray]
+  def getEntriesType(rvRowType: PStruct): PStruct = rvRowType.selectFields(Set(entriesIdentifier), keep = true)
```

Comment:
`getEntriesType` seems weird.  It's the row with just the single field entries?  I assume that's for read/write split?  I don't think it should be in MatrixType.

Reply:
Or it needs a clearer name, like `getEntriesOnlyRowType`.